### ISSUE TITLE
[BUG] Request with empty list of fields crash

### ISSUE THREAD

**Issue by theSlyest** [GodotNuts/GodotFirebase#415]
I'm trying to make a query where I only need document Ids, so I put an empty list of fields in the `select()` filter. Example:
```
query.select([]).from("origins", false).where("name", FirestoreQuery.OPERATOR.EQUAL, origin).limit(1)
```
Then I have an error in firestore_document.gd:24
```
document = doc.fields
```
The equivalent web request works.

**Comment by theSlyest** [comment]
I solved this error and would gladly commit the simple solution.

**Comment by WolfgangSenff** [comment]
Awesome! Go ahead and fork whatever branch you're using then and make a PR for it! Thanks!

**Comment by WolfgangSenff** [comment]
Solved by your very own:  #417 !
